DesignSafe-CI/portal
# chore: missing db migration for #1397

## Overview: ##

Add migration for #1397.

## PR Status: ##


## Related Jira tickets: ##

* [WA-324](https://tacc-main.atlassian.net/browse/WA-324)

## Summary of Changes: ##

- **added** migration for `ordering`

## Testing Steps: ##

1. Checkout branch.
2. Run CMS.
    <sup>`make start`</sup>
3. Enter CMS container.
    <sup>`docker exec -it des_django bash`</sup>
4. [Run migration.](https://docs.djangoproject.com/en/5.0/ref/django-admin/#migrate)
    <sup>`./manage.py migrate workspace 0017_alter_applistingentry_options_ordering`</sup>
5. Verify no error upon migrate.
    - No error in command output.
    - (Optional) No error loading app listing entry.
        <sup>If you have an app listing entry, visit it e.g. [/admin/.../1/change/](https://designsafe.dev/admin/workspace/applistingentry/1/change/).</sup>
6. (Optional) [Reverse migration.](https://docs.djangoproject.com/en/5.0/topics/migrations/#reversing-migrations)
    <sup>`./manage.py migrate workspace 0016`</sup>

## UI Photos:

Skipped.

## Notes: ##

> [!IMPORTANT]
> Do migrations get made and run on deploy? I will be opening a PR with a model change, and I want to know whether I should make migrations for it in that PR.
